apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -148,6 +149,39 @@ static wifi_manager_cb_s g_wifi_callbacks = {
 	_wt_scan_done,
 };
 
+/* Supported security method */
+static const char* g_wifi_test_auth_method[] = {
+#ifdef WT_AUTH_TABLE
+#undef WT_AUTH_TABLE
+#endif
+#define WT_AUTH_TABLE(type, str, desc) str,
+#include "wm_test_auth_table.h"
+};
+
+static const wifi_manager_ap_auth_type_e g_auth_type_table[] = {
+#ifdef WT_AUTH_TABLE
+#undef WT_AUTH_TABLE
+#endif
+#define WT_AUTH_TABLE(type, str, desc) type,
+#include "wm_test_auth_table.h"
+};
+
+static const char* g_wifi_test_crypto_method[] = {
+#ifdef WT_CRYPTO_TABLE
+#undef WT_CRYPTO_TABLE
+#endif
+#define WT_CRYPTO_TABLE(type, str, desc) str,
+#include "wm_test_crypto_table.h"
+};
+
```

Comment:
Please remove g_wifi_test_crypto_method, WT_CRYPTO_TABLE
as it is already present in wm_test_utils.c

Reply:
Done